2019-04-07 14:37:48 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Admin/eclipse-workspace/TestH2/lib/base1.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Admin/eclipse-workspace/TestH2/lib/base1.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at Conexion.<init>(Conexion.java:17)
	at Sistema.main(Sistema.java:6)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Admin/eclipse-workspace/TestH2/lib/base1.mv.db [1.4.199/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Admin/eclipse-workspace/TestH2/lib/base1.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 26 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Admin/eclipse-workspace/TestH2/lib/base1.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 20 more
2019-04-07 16:54:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "SCHEMA" no encontrada
Table "SCHEMA" not found; SQL statement:
delete schema prueva [42102-199]
2019-04-07 16:55:28 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PRUEVA" debido a que "USUARIOS, PRIMARY_KEY_3, CONSTRAINT_3" depende de él.
Cannot drop "PRUEVA" because "USUARIOS, PRIMARY_KEY_3, CONSTRAINT_3" depends on it; SQL statement:
drop schema prueva [90107-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:573)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropSchema.update(DropSchema.java:63)
	at org.h2.command.CommandContainer.update(CommandContainer.java:133)
	at org.h2.command.Command.executeUpdate(Command.java:267)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:233)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1113)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1436)
	at org.h2.server.web.WebApp.query(WebApp.java:1111)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1073)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1060)
	at org.h2.server.web.WebThread.process(WebThread.java:173)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
2019-04-07 18:51:45 jdbc[5]: null
org.h2.message.DbException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:242)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:182)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:299)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:782)
	at org.h2.server.web.WebApp.login(WebApp.java:1004)
	at org.h2.server.web.WebApp.process(WebApp.java:219)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 15 more
2019-04-07 19:23:58 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "COCHES" no encontrada
Table "COCHES" not found; SQL statement:
ALTER TABLE coches ADD CONSTRAINT coches_pk PRIMARY KEY ( id_coche ) [42102-199]
2019-04-07 19:31:44 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "PRUEVA" no encontrada
Table "PRUEVA" not found; SQL statement:
drop table prueva [42102-199]
2019-04-07 19:31:59 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PRUEVA" debido a que "USUARIOS, PRIMARY_KEY_3, CONSTRAINT_3" depende de él.
Cannot drop "PRUEVA" because "USUARIOS, PRIMARY_KEY_3, CONSTRAINT_3" depends on it; SQL statement:
drop schema prueva [90107-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:573)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropSchema.update(DropSchema.java:63)
	at org.h2.command.CommandContainer.update(CommandContainer.java:133)
	at org.h2.command.Command.executeUpdate(Command.java:267)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:233)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1113)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1436)
	at org.h2.server.web.WebApp.query(WebApp.java:1111)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1073)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1060)
	at org.h2.server.web.WebThread.process(WebThread.java:173)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
2019-04-07 20:01:19 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "ROKER_MOTORS" no encontrado
Schema "ROKER_MOTORS" not found; SQL statement:
insert into roker_Motors.usuarios values (1,'Guillermo','Villanueva Pedraza','devott@fairmail.com','123456') [90079-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:573)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.getSchema(Parser.java:1132)
	at org.h2.command.Parser.getSchema(Parser.java:1138)
	at org.h2.command.Parser.readTableOrView(Parser.java:7122)
	at org.h2.command.Parser.readTableOrView(Parser.java:7117)
	at org.h2.command.Parser.parseInsert(Parser.java:1682)
	at org.h2.command.Parser.parsePrepared(Parser.java:891)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1113)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1436)
	at org.h2.server.web.WebApp.query(WebApp.java:1111)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1073)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1060)
	at org.h2.server.web.WebThread.process(WebThread.java:173)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
